scale/settings.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/python3
+
+""" Shared settings and constant values across multiple scaling scripts"""
+
+# Deprecated constant context values used by Kubernetes
+# NAMESPACES = ['datahub', 'prob140', 'stat28']
+# CLUSTER = 'prod'
+# KUBECTL_CONTEXT = 'gke_data-8_us-central1-a_prod'
+GCLOUD_INSTANCE_GROUP = 'gke-dev-default-pool-dbd2a02e-grp'
+GCE_ZONE = "us-central1-a"
+
+# Legacy scaling settings
+USERS_PER_NODE = 7
+#POD_THRESHOLD = 0.9
+#BUMP_INCREMENT = 2
+
+# New scaling settings #TODO: discuss with BIDS
+CAPACITY_PER_NODE = USERS_PER_NODE  # FIXME: should be determined dynamically
```

Comment:
@yuvipanda @ryanlovett

It is possible to get the quota limit for the pod (the container) and use the **limit** to determine the workload. (@SaladRaider Please let me know how resource quota is done in the current deployment) The capacity of node might also need to be dynamically determined using node model.

It would however be a much more aggressive scaling policy, if I use the available RAM to determine workload. Will there be a surge of memory usage when a user starts to run a script with a big dataset? Worst case scenario, swap (if configured) will be used by docker. Is that acceptable?